src/components/image-upload/index.tsx

```diff
@@ -11,32 +15,60 @@ export interface Properties {
 }
 
 interface State {
-  files: any;
+  files: File[];
 }
 
 export class ImageUpload extends Component<Properties, State> {
   state = { files: [] };
+  fileInputRef = React.createRef<HTMLInputElement>();
 
-  onDrop = (files) => {
+  onDrop = (files: File[]) => {
     this.setState({ files });
 
     if (typeof this.props.onChange === 'function') {
       this.props.onChange(files[0]);
     }
   };
 
-  renderImage = () => {
+  onSimulateClick = () => {
+    if (this.fileInputRef.current) {
+      this.fileInputRef.current.click(); // simulating a click on the file input when the button is clicked
+    }
```

Comment:
The `onClick` on the `inputProps` is meant to simulate a click on the file input, which opens the file dialog for selecting a file. As a result, we can add the same behaviour by using a reference to the input element and programmatically clicking it.